src/plugins/index_pattern_management/kibana.json

```diff
@@ -3,6 +3,6 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["management", "data", "kibanaLegacy"],
+  "requiredPlugins": ["management", "data", "kibanaLegacy","home"],
```

Comment:
I might have missed this elsewhere in the PR, but what is `home` required for? I don't see it being used in `plugin.ts` or in `IndexPatternManagementSetupDependencies`